pkgs/development/python-modules/hiredis-py/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv
+, fetchurl
+, buildPythonPackage
+}:
+buildPythonPackage rec {
```

Comment:
Not sure whether you need `rec` here.

Reply:
I had removed `rec` and split adding myself to maintainers into another commit.